xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -13,8 +13,14 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>The base class of web encoders.</summary>
+    <remarks>
+    <format type="text/markdown"><![CDATA[ 
+
+## Remarks  
+ TextEncoder subclasses can be used to do HTML encoding, URI encoding, and JavaScript encoding. Instances of such subclasses can be accessed using the <xref:System.Text.Encodings.Web.HtmlEncoder.Default>, <xref:System.Text.Encodings.Web.UrlEncoder.Default>, and <xref:System.Text.Encodings.Web.JavaScriptEncoder.Default> methods.  
```

Comment:
Each of these xrefs needs to include `?displayProperty=fullName`
Default methods --> Default properties